pkg/runtime/containerd/client.go

```diff
@@ -376,7 +381,26 @@ func (cc *ctdClient) RunContainer(image meta.OCIImageRef, config *runtime.Contai
 	snapshotter := cc.client.SnapshotService(containerd.DefaultSnapshotter)
 
 	// Add the /etc/resolv.conf mount, this isn't done automatically by containerd
-	config.Binds = append(config.Binds, runtime.BindBoth("/etc/resolv.conf"))
+	// Ensure a resolv.conf exists in the vmDir. Calculate path using the vm id
+	// TODO(stealthybox):
```

Comment:
This is here for posterity, but I can move it into an issue or just remove it altogether depending on what we want.